enterprise/frontend/src/embedding-sdk/cli/constants/messages.ts

```diff
@@ -35,11 +39,17 @@ export const getEmbeddingFailedMessage = (reason: string) => `
   Reason: ${reason}
 `;
 
-export const getMetabaseInstanceSetupCompleteMessage = (instanceUrl: string) =>
+export const getMetabaseInstanceSetupCompleteMessage = (
+  instanceUrl: string,
+  email: string,
+  password: string,
+) =>
   // eslint-disable-next-line no-unconditional-metabase-links-render -- link for the CLI message
   `
   Metabase is running at ${blue(instanceUrl)}
-  You can find your login credentials at METABASE_LOGIN.json
+
+  Login with email "${blue(email)}" and password "${blue(password)}".
+  You can find your login credentials at "${blue(SAMPLE_CREDENTIALS_FILE_NAME)}".
```

Comment:
```suggestion
  You can also find your login credentials at "${blue(SAMPLE_CREDENTIALS_FILE_NAME)}".
```

Since we output user name and password. This sentence is an additional info.